examples/example_mineral.py

```diff
@@ -61,7 +61,7 @@
 
 
 input_filename = 'avng.jpl.nasa.gov/AVNG_2015_data_distribution/L2/ang20150420t182050_rfl_v1e/ang20150420t182050_corr_v1e_img.hdr'
-library_filename='../pycoal/tests/s06av95a_envi.hdr'
+library_filename='../pycoal/tests/s07_AV95_envi.hdr'
```

Comment:
You need to make sure that there are no regressions what-so-ever. We are NOT dropping support for USGS v6. We are _**adding**_ functionality to use v7... this means that all v6 code should not be touched.
The solution here is to either

1. introduce a **_brand new example file_** for v7 mineral classification, or 
1. introduce a new CLI flag which determines the spectral library at runtime based upon the program input.